### DISCUSSION TITLE
How to Disable IPv6 on Monitoring Interface

### DISCUSSION THREAD

**Discussion by blackbrownco** [Security-Onion-Solutions/securityonion#6256] [Unsupported Versions]
Hi,

I am using Security Onion 2.3.80 with ISO install. 
My Kibana shown either source.ip or destination.ip with IPv6. Therefore, there is no record shown with IPv6 within Hunt interface. Is there any way to disable the IPv6 since the machines that being monitored by SecOnion are using IPv4. 

I've also checked on /etc/sysctl.conf and all the values are set to 1
````bash
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
````

Thanks.

**Comment by dougburks** [comment]
> My Kibana shown either source.ip or destination.ip with IPv6. Therefore, there is no record shown with IPv6 within Hunt interface.

I'm not sure what you mean here. Kibana and Hunt are totally separate interfaces but both should query Elasticsearch and display results in the same way.
